cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsSessionCredentialClient.java

```diff
@@ -95,8 +90,17 @@ private AwsSessionCredentials getAwsSessionCredentialsAndAssumeRole(AwsCredentia
         }
     }
 
-    private AmazonSecurityTokenServiceClient awsSecurityTokenServiceClient(AwsCredentialView awsCredential) {
-        return awsClient.createCdpSecurityTokenServiceClient(awsCredential);
+    private AWSSecurityTokenService awsSecurityTokenServiceClient(AwsCredentialView awsCredential) {
+        if (!awsEnvironmentVariableChecker.isAwsAccessKeyAvailable(awsCredential)
+                || !awsEnvironmentVariableChecker.isAwsSecretAccessKeyAvailable(awsCredential)) {
+            LOGGER.debug("AWSSecurityTokenServiceClient will use aws metadata because environment variables are undefined");
```

Comment:
this is not necessary anymore we discussed with @lacikaaa

Reply:
I was not part of that discussion, but this is a simple revert.

Reply:
I can't remember but sure :D

Reply:
Logging removed